docs/gambit/gambit.md

```diff
@@ -112,7 +112,7 @@ We recommend this approach only when you have a simple project with few files
     you provide Solidity's [allowed paths][allowed] to `solc` using the `--allow-paths` argument.
```

Comment:
Provide Solidity's [allowed paths][allowed] to `solc` using the `--allow-paths` argument.